src/Common/tests/System/Net/Capability.Version.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Runtime.InteropServices;
+
+namespace System.Net.Test.Common
+{
+    public static partial class Capability
+    {
+        // TODO: Using RtlGetVersion is temporary until issue #4741 gets resolved.
+        [DllImport("ntdll", CharSet = CharSet.Unicode)]
+        private static extern int RtlGetVersion(ref RTL_OSVERSIONINFOW lpVersionInformation);
+
+        [StructLayout(LayoutKind.Sequential, CharSet = CharSet.Unicode)]
+        private struct RTL_OSVERSIONINFOW
+        {
+            internal uint dwOSVersionInfoSize;
+            internal uint dwMajorVersion;
+            internal uint dwMinorVersion;
+            internal uint dwBuildNumber;
+            internal uint dwPlatformId;
+            [MarshalAs(UnmanagedType.ByValArray, SizeConst = 128)]
+            internal char[] szCSDVersion;
+        }
+
+        public static bool IsWindows10Platform()
```

Comment:
Sorry, I wasn't clear: I thought the capability would be `SocketsReuseUnicastPortSupport` and would include checks for everything required, not only the OS version.